doc/test-framework/scorecard.md

```diff
@@ -49,6 +49,7 @@ Following are some requirements for the operator project which would be  checked
 1. Setup the `.osdk-scorecard.yaml` configuration file in your project.. See [Config file](#config-file)
 2. Create the namespace defined in the RBAC files(`role_bindinding`)
 3. Then, run the scorecard, for example `$ operator-sdk scorecard`. See the [Command args](#command-args) to check its options.
+4. Note, starting with version v1alpha2, the scorecard return code is 1 if any of the tests executed did not pass and 0 if all selected tests pass.
 
```

Comment:
I think it is not required here.

Reply:
Maybe add some NOTE: as if you use version 2 then the output will be 0 or 1 to indicate that the test pass or fails respectively.  But, IMHO not here. It could be done where has the detailed info over its results/outputs.

Reply:
I could make it a separate section like man pages do, something like a heading of EXIT STATUS if that makes it clearer.  My intent here is to let the end user know what the exit codes will be out of scorecard so I figured this was the correct document for that communication.

Reply:
It is fine. But this section is describing the 3 macro steps to set up the scorecard, so really make sense the 4 step is a description of the result when the v2 is used?

Reply:
I added an Exit Status heading to call this out separately.